Sources/SwiftDocC/Semantics/Metadata/Availability.swift

```diff
@@ -92,7 +102,7 @@ extension Metadata {
 
         /// The platform version that this page applies to.
         @DirectiveArgumentWrapped
-        public var introduced: String
+        public var introduced: VersionTriplet
```

Comment:
Note: This change is source breaking.

Reply:
I've tried to fix this so there's no longer a source breaking change: c2b2b9ce0a2b135232e49fa9dee84bf7d86f9cea